toolkit/tools/pkg/imagecustomizerlib/testdata/http-app.yaml

```diff
@@ -0,0 +1,99 @@
+os:
```

Comment:
do we need this content in both this file and the .md page?

Reply:
Having a copy in a test YAML is handy for manual testing.